keyboards/fancytech/fancyalice66/config.h

```diff
@@ -0,0 +1,12 @@
+// Copyright 2022 chent7 (@chent7)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#ifdef RGB_MATRIX_ENABLE
+    #define RGB_MATRIX_LED_COUNT 82
+
```

Comment:
```suggestion
#    define RGB_DI_PIN B0
#    define RGB_MATRIX_LED_COUNT 82
```

Reply:
At this point, to facilitate migration to ws2812 pin config.
```suggestion
#    define WS2812_DI_PIN B0
#    define RGB_DI_PIN WS2812_DI_PIN
#    define RGB_MATRIX_LED_COUNT 82

```